### PR TITLE
[bullet3] Install CMake toolchain files and targets

### PR DESCRIPTION
**Pull request by stephenjust**
Install CMake toolchain files and targets

* Add CMake targets for each bullet library
* Install `BulletConfig.cmake` (and other files) to `{prefix}/share/bullet3`

This fixes the behavior of `find_package(bullet {version})` and `target_link_libraries`.

Prior to this patch, library consumers would need to specially handle debug compilation where the Windows library files were suffixed by `_Debug`.
Now, `target_link_libraries(target PRIVATE "${BULLET_LIBRARIES}")` will resolve correctly in both Debug and Release builds.

### CODE REVIEW

File: ports/bullet3/portfile.cmake [hunk 1]
```diff
@@ -32,8 +33,19 @@ vcpkg_configure_cmake(
 
 vcpkg_install_cmake()
 
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/lib/cmake)
-file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/lib/cmake)
+# Move debug cmake toolchain files
+file(COPY ${CURRENT_PACKAGES_DIR}/debug/share/bullet3/ DESTINATION ${CURRENT_PACKAGES_DIR}/share/bullet3 PATTERN *.cmake )
+
+# Fix debug targets
+file(GLOB DebugTargetFiles "${CURRENT_PACKAGES_DIR}/share/bullet3/*_Export-debug.cmake")
```
Comment (cenit) [inline ports/bullet3/portfile.cmake]: If you use `-Targets` insead of `-Export` as the exports filename suffix (it requires some changes in your pr), then you can call

```vcpkg_fixup_cmake_targets(...)```

which will fix this and other small mismatches for you

Comment (stephenjust) [reply ports/bullet3/portfile.cmake]: Good catch, I've made the suggested change.

**Comment by PhoebeHui** [comment]
@stephenjust, thanks for the PR!

Could you also update the version in  bullet3 CONTROL file? it should be 'Version: 2.88-2'.

**Comment by PhoebeHui** [comment]
bullet3 feature 'multithreading' test pass on x86-windows.

**Review by PhoebeHui — APPROVED** [review]

**Review by dan-shaw — APPROVED** [review]

**Comment by dan-shaw** [comment]
LGTM, waiting for macOS CI pipelines to be back up.

**Pull request merged**